src/SAML2/AuthnRequest.php

```diff
@@ -153,14 +219,20 @@ public function __construct(DOMElement $xml = NULL)
                 $this->nameIdPolicy['AllowCreate'] = SAML2_Utils::parseBoolean($nameIdPolicy, 'AllowCreate', FALSE);
             }
         }
+    }
 
+    /**
+     * @param DOMElement $xml
+     */
+    protected function parseRequestedAuthnContext(DOMElement $xml)
+    {
         $requestedAuthnContext = SAML2_Utils::xpQuery($xml, './saml_protocol:RequestedAuthnContext');
         if (!empty($requestedAuthnContext)) {
             $requestedAuthnContext = $requestedAuthnContext[0];
 
             $rac = array(
                 'AuthnContextClassRef' => array(),
-                'Comparison' => 'exact',
+                'Comparison'           => 'exact',
```

Comment:
It's only used in the RequestedAuthnContext, which so far we only support for the AuthnRequest.